distsql/select_result.go

```diff
@@ -325,6 +325,7 @@ type selectResultRuntimeStats struct {
 	totalProcessTime time.Duration
 	totalWaitTime    time.Duration
 	rpcStat          tikv.RegionRequestRuntimeStats
+	CoprCacheHit     int64
```

Comment:
How about renaming this field to `CoprCacheHitNum` or `CoprCacheNumHit`?
`selectResultRuntimesStats.CoprCacheHit` and `CopRuntimeStats.CoprCacheHit` having the same name but different types seems ambiguous.